data/remnant missions.txt

```diff
@@ -495,3 +560,703 @@ mission "Remnant: Technology Available"
+			`	"Thank you," he signs. "It is acting differently, jumping around destroying ships instead of looting and running. If it simply looted and left we would be pleased, but now it is learning about us instead, so it has reached the end of its usefulness. Please disable it so that our technicians can get on board." He pauses, then continues, "I doubt we'll be able to salvage the ship itself - they pack those things with soldiers - but we should be able to strip everything else of use off it."`
+				accept
+	npc save disable
+		government Korath
+		personality heroic vindictive target uninterested waiting
+		system
+			distance 1 2
+		fleet
+			names "korath"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have disabled the Korath ship that was left over from the raid on <planet>. You can now return there to collect your payment."
+	on fail
+		payment 250000
+		dialog `You are met by the prefect who thanks who looks upset. "The ship was supposed to be disabled so its contents could be salvaged!" he signs angrily. Nonetheless, he shoves a credit chip worth <payment> into your hand. "That's half what you would have earned if the ship had been disabled." He pauses and shakes his head. "I think Tali might have some more work for you in the spaceport, but in the meantime, why don't you go find some Korath to practice disabling? It is a valuable skill among the Remnant."`
+	on complete
+		payment 500000
+		dialog `You are met by the prefect who thanks you for disabling it and hands you a credit chip for <payment>. "I think Tali might have some more work for you in the spaceport, but in the meantime, don't forget to drop off any salvaged outfits in the outfitters so the researchers can look at them."`
+
+
+
+mission "Remnant: Bounty 2"
+	job
+	repeat
+	name "Hunt down the <npc>"
+	description "Hunt down a Korath ship that is lurking in Remnant territory, then return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Salvage 4: offered"
+		random < 60
+	npc kill
+		government "Korath"
+		personality heroic vindictive target uninterested waiting
+		system
+			distance 1 2
+		fleet
+			names "korath"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have destroyed the Korath ship. You can now return to <planet> to collect your payment."
+	on complete
+		payment 300000
+		dialog "A Remnant military leader thanks you for hunting down the Korath ship <npc>, and gives you the agreed-upon payment of <payment>."
+
+
+
+mission "Remnant: Salvage 5"
+	name "Salvage Delivery"
+	description `Deliver some high value salvage to Caelian.`
+	source "Viminal"
+	cargo "Korath Salvage" 20
+	blocked `Tali would like you to deliver some valuable Korath equipment to Caelian, but you will need 20 tons of space to carry it.`
+	destination "Caelian"
+	to offer
+		or
+			has "Remnant: Salvage 4: done"
+			has "Remnant: Salvage 4: failed"
+		has "Remnant: Learn Sign 3: done"
```

Comment:
Remove this line, as 1) it isn't a valid condition anymore and 2) there shouldn't be a way to stop this mission string mid-way through without any direction as to how to continue it.